src/gui/properties/peerlistwidget.cpp

```diff
@@ -90,6 +90,9 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
     setEditTriggers(QAbstractItemView::NoEditTriggers);
     setSelectionMode(QAbstractItemView::ExtendedSelection);
     header()->setStretchLastSection(false);
+	header()->setTextElideMode(Qt::ElideRight);
+	setTextElideMode(Qt::ElideRight);
```

Comment:
This is not needed anywhere because it's already the default. Also fix indentation everywhere so it is in line with the neighboring lines.